rke2/konkong/rook-ceph/rook-config-override.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: rook-config-override
+  namespace: rook-ceph
+data:
+  config: |
+    [mon]
+    mon_data_avail_warn = 20
```

Comment:
I see that this got committed for pillan too but it shouldn't be needed with the new partition table giving the mon it's own filesystem.